README.md

```diff
@@ -28,7 +28,7 @@ Assumptions:
 
 4. Switch to the alpha release version of the code. This is the latest stable release.
```

Comment:
This could say "Switch to the master branch of the code. This is the latest stable release."